traffic_monitor/cache/cache_test.go

```diff
@@ -95,3 +100,36 @@ func TestComputeStatGbps(t *testing.T) {
 		}
 	}
 }
+
+func TestParseAndDecode(t *testing.T) {
+	file, err := ioutil.ReadFile("stats_over_http.json")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	pl := &poller.HTTPPollCtx{HTTPHeader: http.Header{}}
+	ctx := interface{}(pl)
+	ctx.(*poller.HTTPPollCtx).HTTPHeader.Set("Content-Type", rfc.ApplicationJSON)
```

Comment:
There's also a constant for `"Content-Type"`

Reply:
That said, hard-coding constants in tests is good because it catches regressions if the library constants are ever changed. Though this is @ocket8888's suggestion, so up to them